.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,25 +12,30 @@
 Add one of the following:
 bug
 cleanup
+dependency update
 documentation
 feature
+improvement
+release workflow
+testing
 -->
 
-**Which issue does this PR fix**:
+**Which issue does this PR fix?**:
+<!-- If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue -->
 
 
-**What does this PR do / Why do we need it**:
-
-
-**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:
-
+**What does this PR do / Why do we need it?**:
 
 **Testing done on this change**:
 <!--
 output of manual testing/integration tests results and also attach logs
```

Comment:
Sorry, one last nit while we are here. Can we update this comment to be more sentence-like? Maybe:
```
Please paste the output from manual and/or integration test results. Please also attach any relevant logs.
```